packages/codelabs/src/basics/lit-html.md

```diff
@@ -22,21 +22,19 @@ In this codelab, you will learn the basics of building web components with lit-h
 - Conditional rendering
 - Handling events
 - Managing data
-- Creating child components
+- Creating encapsulated components
```

Comment:
```suggestion
- Creating child components
```
While its true that they're encapsulated, I think in the context of the codelab it makes more sense to say 'child' component, to demonstrate how parent/child relations work